tests/topotests/static_routing_with_ibgp/test_static_routes_topo4_ibgp.py

```diff
@@ -0,0 +1,1213 @@
+                            addr_type
+                        ].split("/")[0]
+                        + "/16",
+                        "next_hop": topo["routers"]["r2"]["links"]["r3-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                ]
+            }
+        }
+
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+
+        # Enable static routes
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": topo["routers"]["vm4"]["links"]["r1"][
+                            addr_type
+                        ].split("/")[0]
+                        + "/16",
+                        "next_hop": topo["routers"]["r1"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                    {
+                        "network": topo["routers"]["vm5"]["links"]["r3"][
+                            addr_type
+                        ].split("/")[0]
+                        + "/16",
+                        "next_hop": topo["routers"]["r3"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                    {
+                        "network": topo["routers"]["vm4"]["links"]["r1"][addr_type],
+                        "next_hop": topo["routers"]["r1"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                    {
+                        "network": topo["routers"]["vm5"]["links"]["r3"][addr_type],
+                        "next_hop": topo["routers"]["r3"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                    {
+                        "network": topo["routers"]["r1"]["links"]["vm4"][addr_type],
+                        "next_hop": topo["routers"]["r1"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                    {
+                        "network": topo["routers"]["r3"]["links"]["vm5"][addr_type],
+                        "next_hop": topo["routers"]["r3"]["links"]["r2-link0"][
+                            addr_type
+                        ].split("/")[0],
+                    },
+                ]
+            }
+        }
+
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+        sleep(10)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.